docs/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,192 @@
+.. _laravel-tls:
+
+========================
+Enable and Configure TLS
+========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, security, connection options
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the TLS protocol to secure 
+your connection to a MongoDB deployment. To configure your connection to
+use TLS, enable the TLS option and provide your certificates for
+validation when configuring a connection in your application's
+``config/database.php`` file.
+
+.. tip::
+   
+   To learn more about TLS, see the Wikipedia entry on
+   :wikipedia:`Transport Layer Security <w/index.php?title=Transport_Layer_Security&oldid=1184063676>`.
+
+Enable TLS
+----------
+
+In your application's ``config/database.php`` file, you can enable TLS
+on a connection to your MongoDB deployment in one of the following ways:
+
+- Setting the ``tls`` option to ``true`` in your connection string
+- Setting the ``tls`` option to ``true`` in the ``options`` property
+
+Select from the following :guilabel:`Connection String` and
+:guilabel:`URI Options` tabs to see a corresponding code sample:
```

Comment:
Suggestion:
I think URI Options might be valid term, but since these are represented as an "options" array in the config file, maybe it could be described as "connection options", which is similar to how they're categorized on the [Connection Options](https://github.com/mongodb/laravel-mongodb/blob/4.3/docs/fundamentals/connection/connection-options.txt#L49-L61) page.